pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -135,15 +141,17 @@ class RichProgressBar(ProgressBarBase):
 
     def __init__(
         self,
-        refresh_rate: float = 1.0,
+        refresh_rate: int = 1,
+        refresh_per_second: float = 1.0,
```

Comment:
@kaushikb11 I've split the refresh rates into two separate variables, however unsure how this will actually perform.